src/Network/Network/Az.Network.psd1

```diff
@@ -355,12 +421,18 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Release for Azure Stack 2020-09-01-hybrid profile.'
+        ReleaseNotes = '* Added Cmdlets for Identity on Application Gateway.
```

Comment:
Since the new version will be released based on 1.2.1 or 1.2.0, the release note should be different from them. Notes should be sign assemblies ...

Another approach is that you submit 2 PR. The first one is just to sync code of 1.2.0 or 1.2.1. Then the second one is bump version and change release notes for 1.2.2